### PR TITLE
Design sprint blog post

### PR DESCRIPTION
**Pull request by stevenjmesser**
Wrote a weeknote about what we did this week. Comments and suggestions _very_ welcome.

(Also fixed the summary on a previous post.)

### CODE REVIEW

File: content/blog-post/jamming-together.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+A design sprint is a five-day process for building out and testing ideas to solve problems for our users. It’s close, collaborative work. Focused time away from the usual routine, away from Teams meetings and Slack, allowing us to go deeper, together. It’s a bit like being in a band: we get to jam together. 
+
+## Understanding the problems
+
+It all started on Monday. We got together to talk about the challenge, and we mapped out how people working at LPAs provide data currently – including all the [invisible parts](https://good.services/blog/the-problem-of-invisible-services) of the process. As we learned from an anthropology study done by [Projects by IF](https://www.projectsbyif.com/), there’s several people working at LPAs involved in the process:
+
+- people leading the way in improving planning or digital ways of working
+- people driving project delivery
+- people on the ground within planning, and
```
Comment (frankieroberto) [inline content/blog-post/jamming-together.md:20]:
```suggestion
- people on the ground within planning
```

File: content/blog-post/jamming-together.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- people managing geospatial data.
```
Comment (frankieroberto) [inline content/blog-post/jamming-together.md:21]:
```suggestion
- people managing geospatial data
```

File: content/blog-post/jamming-together.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+We talked about the barriers they face providing data, related to tech, the data itself, how different organisations are set up and contextual factors that impact the organisations. We also called out some problems that users have mentioned to us or that we’ve experienced over the last twelve months operating the platform.
+
+## Identifying opportunities
+
+Standing back, looking at all those together, we reframed the problems as opportunities for making things better. There were three main clusters:
+
+- One cluster focused on how we might make the journey of providing data more complete, an [‘end-to-end service’](https://www.gov.uk/service-manual/design/introduction-designing-government-services#the-characteristics-of-a-good-government-service). That means meeting users where they are with contextual guidance, and breaking the process down into chunks that can be completed in an agile, iterative way. We also highlighted being better at communicating progress or specific processes to users.
```
Comment (frankieroberto) [inline content/blog-post/jamming-together.md:29]: This could be regular paragraphs rather than a list?

**Review by frankieroberto — COMMENTED** [review]
This is great!

Some minor style suggestions.

A photo or two would be be helpful I think! Show your wall of post-its? Or a screenshot of your prototype?

**Comment by psd** [comment]
Agree , some visuals will help. Thanks for your feedback Frankie, really helpful. I think it was a great week. I felt like we didn't so much learn the things listed, rather quickly proved they were good ideas by mocking them up and getting them in front of users, all in a week, which was impressive and transformative to how the team feels about their work, which has been hard and sow. Chapeau!

**Comment by stevenjmesser** [comment]
> I felt like we didn't so much learn the things listed, rather quickly proved they were good ideas

V true!

**Pull request merged**